src/en/tut-lxd.md

```diff
@@ -0,0 +1,264 @@
+Title: Getting started with Juju and LXD
+
+# Getting started with Juju and LXD
+
+[LXD](lxd-upstream) provides an amazing experience using machine containers to be able to
+create the same models that are created with Juju on other public and private
+clouds. Using LXD as a method to test, verify, and replicate complex software
+deployments is a powerful tool that every Juju user needs. These instructions
+will get you up and running and deliver the best-possible experience with
+Juju. At the moment, that means using the latest release of Ubuntu with '[Long
+Term Support][long-term-support]' (LTS): [16.04 LTS
+                                          (Xenial)][Xenial-download].
+
+Our system will need the following:
+
+- [LXD][LXD-upstream]: a hypervisor for LXC, providing fast, secure containers.
+- [ZFS][ZFS-wiki]: a highly efficient and feature-rich filesystem and logical volume manager.
+
+
+## Install the software
+
+Begin by installing the required software. You'll need Juju and LXD on the
+machine. This means that using LXD is not supported on the breadth of Juju
+supported operating systems.
+
+For more options in installing Juju see the [install docs][install].
+
+
+```bash
+sudo snap install --classic juju
+sudo apt install juju lxd zfsutils-linux
+```
+
+## Groups and LXD initialisation
+
+In order to use LXD, your user must be a member of the `lxd` group.  This
+should already be the case but you can confirm this by running the command:
+
+```bash
+groups
+```
+
+Your groups may vary, but if `lxd` is absent you can add yourself to the `lxd`
+group with:
+
+```bash
+newgrp lxd
+```
+
+Secondly, LXD includes an interactive initialisation which includes setting up
+a ZFS pool and appropriate networking for your LXD containers. To start this
+process, enter:
+
+```bash
+sudo lxd init
+```
+
+You will be asked several questions to configure LXD for use. Pressing Enter
+will accept the default answer (provided in square brackets). Only one answer,
+the size of the loop device in the below example, uses a non-default value.
```

Comment:
I don't know if it is worth mentioning, but the default value for the loop device size is now dynamic (it is based on a % of storage available). I think it probably isn't worth it, but what do you think?